README.rst

```diff
@@ -1,15 +1,18 @@
 ================
 ScanCode toolkit
 ================
+
 A typical software project often reuses hundreds of third-party packages.
 License and origin information is not always easy to find and not normalized:
 ScanCode discovers and normalizes this data for you.
 
+Read more about ScanCode here: `scancode-toolkit.readthedocs.io <https://scancode-toolkit.readthedocs.io/en/latest/>`_.
+
 Why use ScanCode?
-=======================
+=================
 
 - As a **standalone command line tool**, ScanCode is **easy to install**, run
-  and embed in your CI/CD processing pipeline. It runs on **Windows, macOS and Linux**.
+  and embed in your CI/CD processing pipeline. It runs on **Windows, MacOS and Linux**.
```

Comment:
`macOS` is the correct case from what I can see on Apple's website

Reply:
Oops, will correct this.

Reply:
Done!